src/apollo.js

```diff
@@ -0,0 +1,77 @@
+import { ApolloClient } from '@apollo/client/core'
```

Comment:
This is a general non-Vue apollo client, for use by underlying services and workers.